subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1957,3 +1957,16 @@ static const struct socket_op_vtable tls_sock_fd_op_vtable = {
 	.getsockopt = tls_sock_getsockopt_vmeth,
 	.setsockopt = tls_sock_setsockopt_vmeth,
 };
+
+static bool is_tls(int family, int type, int proto)
```

Comment:
keep the naming convention tls_<supports/check/whatever name will be chosen as noticed in first commit>

Reply:
Well, the naming here is not that critical imho as the function is internal to the file. It feels much more natural for these kind of functions to start with is_xxx()